.github/workflows/pr-title-check.yml

```
@@ -0,0 +1,70 @@
+      - name: Validate PR title
+        id: validate
+        uses: amannn/action-semantic-pull-request@v6.1.1
+        continue-on-error: true
+        with:
+          # Default types from: https://github.com/commitizen/conventional-commit-types
+          requireScope: false
+          wip: true
+
+      - name: Print Failure Documentation
+        if: steps.validate.outcome == 'failure'
+        run: |
```

actionlint (1.7.10)

[error] "github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)